cuml/rf_demo.ipynb

```diff
@@ -504,33 +315,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "0.52618"
+       "0.68542"
       ]
      },
-     "execution_count": 9,
+     "execution_count": 8,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "accuracy_score(cu_rf.predict(X_test.as_matrix()), y_test.to_array())"
+    "accuracy_score(cu_rf.predict(X_test), y_test)"
```

Comment:
This is a very small thing, but I think having the two accuracy values printed one above the other in the same cell might be better for comparison